airflow/config_templates/config.yml

```
@@ -1794,6 +1794,15 @@ webserver:
       type: string
       example: "sha256"
       default: "md5"
+    enable_test_connection:
+      description: |
+        The ability to allow testing connections from UI. By default, it is disabled.
```

Wording suggestion, feel free to take it or not :) 
```suggestion
        Enables or disables the ability to test connections from UI. By default, it is disabled.
```